cmd/cli/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	firestore "pantheon.io/edrt-policy-docs-functions/internal/store"
+)
+
+func main() {
+	fmt.Println("This will be a CLI allowing interactive testing of logic in this repo during development.")
+	fmt.Println("Will probably use a library like cobra to set up the CLI.")
```

Comment:
+1 to using cobra to set up the CLI

Reply:
What do you think about https://github.com/alecthomas/kingpin instead of cobra? 🤔

Reply:
Just for context, I checked through all our repos and found the following:
References to cobra: 391
References to kingpin: 98

So it looks like cobra is the usual choice, but there is precedent for using kingpin. I've never used kingpin before, but I'm happy to have a discussion about it. We should probably settle on one for use across all EDRT repos (whenever a CLI component comes up).